Dockerfile

```diff
@@ -0,0 +1,14 @@
+FROM python:2
+
+RUN apt-get update && apt-get install -y postgresql-client
+
+COPY requirements.txt /usr/src/app/
+WORKDIR /usr/src/app
+RUN pip install -r requirements.txt
+
+EXPOSE 8000
+
+COPY ampcrowd/docker-entrypoint.sh /usr/src/app/ampcrowd
+ENTRYPOINT ["bash", "ampcrowd/docker-entrypoint.sh"]
+CMD ["-d"]
```

Comment:
Ah, didn't realize that about Docker. Yeah, not being in debug mode will run both celery and gunicorn as daemons. Is this a fundamental limitation of Docker? I.e., does this mean it's impossible to run the server as is in production mode on Docker? That surprises me a bit...

I definitely like the `dev.sh` idea.